### ISSUE TITLE
Speaker directory should show past and future speaking engagements

### ISSUE THREAD

**Issue by danroth27** [dotnet-foundation/website#583]
I like the idea of the new .NET speaker directory. It would be great if it was possible to see in a consistent way the speaker's past and future speaker engagements.

**Comment by daveaglick** [comment]
Totally agree that would be awesome - any thoughts on where we could pull that kind of data from? Having individuals provide it is tricky because it'll likely get stale (out of sight, out of mind, etc.) We could consider pulling from Meetup or Sessionize, but I'm not sure how many speakers are on those platforms, and some early discussions revolved around how that makes speakers who aren't look less experienced. Any creative ideas you've got about how to source this data would be _hugely_ helpful.

**Comment by danroth27** [comment]
> Having individuals provide it is tricky because it'll likely get stale (out of sight, out of mind, etc.)

Makes sense. There will need to be some sort of redirection to data that is being kept up to date. 

Pointing to data from Meetup, Sessionize, or YouTube all makes sense to me. It looks like we provide those links already, which is great. It would just be cool if we could also pull to recent and upcoming talks using the data from those services. 

I don't really understand the concern that this would make some speakers seem less experience. If you can't find examples of the speaker speaking, what are you supposed to assume about their experience level?

**Comment by daveaglick** [comment]
> I don't really understand the concern that this would make some speakers seem less experience.

I think the challenge is balancing a few different goals for this directory. One of the obvious goals is to match experienced speakers with folks who need them, and in that context alone any data we can present is a net gain. However, a secondary goal that aligns with the overall mission of the .NET Foundation is to _grow_ experienced speakers in the .NET space and make the overall pool larger by highlighting and promoting new, diverse, and less-experienced (but no less knowledgeable) speakers.

The balance is satisfying the first goal while not totally undermining the second, and the concern is that an overwhelming set of data points that highly favors experience of speakers over knowledge or other metrics will result in a lack of growth because everyone choosing a speaker will gravitate towards those with lots of previous speaking engagements.
